### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by tian-junwei** [openzipkin/zipkin#2176]
What kind of issue is this?
 - [ ] Feature Request. 
hello，in the Zipkin's V2  not support database ？

**Comment by codefromthecrypt** [comment]
can you rephrase? we do have mysql but it is not considered production
level (also indeed it has a v1 style schema)

what database are you looking for?

**Comment by tian-junwei** [comment]
the zipkin's V1 is support mysql，but the zipkin's V2 not support。Do you plan to support mysql in zipkin's V2.

**Comment by codefromthecrypt** [comment]
We don't have plans to change mysql at the moment. It is test only
anyway, so effort isn't important. It is natively implemented with v2
libraries, so you should be fine using it with any recent version of
zipkin and any data format we accept

**Comment by tian-junwei** [comment]
thanks
